.github/workflows/build_and_test.yml

```diff
@@ -121,6 +120,17 @@ jobs:
           cd Tests/Pcap++Test
           if [ -n "$(grep avx512 /proc/cpuinfo)" ]; then Bin/Pcap++Test -n; else echo AVX-512 SUPPORT NOT FOUND, CANNOT RUN Pcap++Test; fi
 
+      - name: Check installation
+        run: cmake --install $BUILD_DIR
+
+      - name: Build Tutorials
+        run: |
+          cmake -DPCAPPP_BUILD_TUTORIALS=ON -S Examples -B build_examples
+          cmake --build build_examples -j
+
+      - name: Test Tutorials
+        run: cd build_examples/tutorials_bin && ./Tutorial-HelloWorld
```

Comment:
We probably don't need that in DPDK CI because `Tutorial-HelloWorld` doesn't use DPDK

Reply:
yes but even if we don't use it, Pcap++ is built with and linking will fail if the lib is not found

That's what I fix in the commit "Fix compilation with PF_RING"